pubspec.yaml

```diff
@@ -3,40 +3,34 @@ description: Mobile app platform for campus.
 version: 1.0.0+1
 publish_to: none
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=3.5.0 <4.0.0"
 dependencies:
-  barcode_widget: 2.0.3
+  barcode_widget: ^2.0.4
   connectivity: 3.0.6
   dio: 4.0.6
   encrypt: 5.0.1
   dots_indicator: 2.1.0
-  firebase_analytics: 8.1.0
-  firebase_core: 1.2.0
-  firebase_crashlytics: 2.0.4
-  firebase_messaging: 10.0.0
+  firebase_analytics: 11.3.3
+  firebase_core: 3.6.0
+  firebase_crashlytics: 4.1.3
+  firebase_messaging: 15.1.3
   flutter:
     sdk: flutter
   flutter_dotenv:
     git:
       url: https://github.com/UCSD/flutter_dotenv.git
       ref: master
-  flutter_scandit_plugin:
-    git:
-      url: https://github.com/UCSD/flutter_scandit.git
-      ref: scandit-6.6.3
-  flutter_linkify: 5.0.2
-  flutter_local_notifications: 9.9.1
+  flutter_scandit_plugin: ^3.0.0
```

Comment:
We may want to update our fork if there is a chance this upstream will go away.

Reply:
Agreed, and also update our fork specifically to target Flutter 3.24 and latest supported iOS/Android versions if possible